nservicebus/sagas/index.md

```diff
@@ -21,6 +21,18 @@ Design processes with more than one remote call to use sagas.
 
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
+## No saga instance available
+
+If a message is received while there is no saga instance, then by default this message gets discarded and succesfully processed. When you have a saga instance that is already completed then it makes sense to discard other messages.
```

Comment:
> gets discarded and succesfully processed

that sounds a bit contradicting. Wouldn't it state the same when just going with `this message gets discarded`?